src/TurbulenceConvection/EDMF_functions.jl

```diff
@@ -780,12 +781,12 @@ function compute_up_tendencies!(
         detr_w = aux_up[i].detr_turb_dyn
         buoy = aux_up[i].buoy
 
-        @. tends_ρaw = -(∇f(wvec(LBC(ρaw * w_up))))
-        @. tends_ρaw +=
-            (ρaw * (I0f(entr_w) * w_en - I0f(detr_w) * w_up)) +
-            (ρ_f * Iaf(a_up) * I0f(buoy)) +
-            nh_pressure
-        tends_ρaw[kf_surf] = 0
+        @. tends_w = -(∇f(wvec(LBC(w * w_up))))
+        @. tends_w +=
+            w * I0f(entr_w) * (w_en - w_up) +
+            I0f(buoy) +
```

Comment:
here I am dividing by `(ρ_f * Iaf(max(a_up, a_min)))` and multiplying again in `nh_pressure`. This should be removed. The main challenge I see here is that in TRMM I `w>0` for `area` slightly above the minimum area for which we filter. The `w` feedbacks through  `nh_pressure` and grows to unphysical values.

Reply:
I forgot that `nh_pressure` is also used in the TKE equation. Once this is done consistently I think the PR is ready